### ISSUE TITLE
(accessibility) conversation buttons should be tab-able from top to bottom

### ISSUE THREAD

**Issue by bcullman** [mckaywrigley/chatbot-ui#175]
When tabbing though the available conversations, they aoear to tab from the bottom up. See

https://user-images.githubusercontent.com/5056972/227752939-2ca04b7d-12ed-4126-a6c8-4733f52debf7.mp4

it would be best if they tab from top down, which is the order they appear.
